Added Social Profiles

aryanguls opened the pull request:
This is to fix issue #42 . 
I have added links to various blogs and social profiles of some of the speakers in the FOSSASIA OpenTechSummit Thailand as well as updated some of the social profiles of the FOSSASIA team.
Apart from this in one of the profiles the Facebook link was being shown twice due to an error in the html file. I have also resolved this issue.
@mariobehling @hpdang Mentors please review the above merge request as and when possible.

aryanguls committed:
Added Social Profiles

mariobehling commented on index.html:
```diff
@@ -702,9 +714,6 @@ <h2>Ivan Ma</h2>
                                             <a target="_self" href="https://www.linkedin.com/in/korakot-chaovavanich-599690121">
                                             <i class="icon social_linkedin"></i>
                                             </a>
-                                            <a href="https://www.facebook.com/korakot.chaovavanich">
```
This seems to be correct. Why do you take it out? Is it wrong?

mariobehling reviewed (changes requested):
Please check comment for line 705.

aryanguls committed:
Update index.html

aryanguls committed:
Update index.html

aryanguls commented:
@mariobehling Sir I had removed that as it had been typed twice this it was showing 2 Facebook profiles for the same account. I this removed one. If you look above that line in the index.html page you can see that the Facebook link is already present there. What I removed was a copy of the same link so that it would be displayed only once on the social profile.

aryanguls commented:
Sir please review.

mariobehling commented:
Thanks. Please help to revert this PR as well: https://github.com/fossasia/fossasia.org/pull/621
